docs/data_flow.md

```
@@ -17,8 +17,9 @@ The "end-to-end" data flow is described there (including Notification Writer ser
 1. `ccx-notification-writer` service stores insights results into AWS RDS database into `new_reports` table.
 1. Content of that table is consumed by `ccx-notification-service` periodically.
 1. Newest results from `new_reports` table is compared with results stored in `reported` table. The records used for the comparison depend on the configured cool down time.
-1. If changes (new issues) has been found, notification message is sent into Kafka topic named `platform.notifications.ingress`. The expected format of the message can be found [here](https://github.com/RedHatInsights/insights-schemas-java/blob/master/insights-notification-schemas-java/src/main/resources/schemas/Action.json)
-1. New issues are also sent to Service Log via REST API. Because Service Log accepts description and content to be represented in Markdown, issues are "rendered" first by [Insights Content Template Renderer](https://github.com/RedHatInsights/insights-content-template-renderer). To use the Service Log API, the `ccx-notification-service` uses the credentials stored in [vault](https://vault.devshift.net/ui/vault/secrets/insights/show/secrets/insights-prod/ccx-data-pipeline-prod/ccx-notification-service-auth).
+1. If changes (new issues) have been found, they are sent to one of two targets depending on the deployment configuration (only one can be active at a time):
+   - **Kafka**: a notification message is sent to the `platform.notifications.ingress` topic. The expected format of the message can be found [here](https://github.com/RedHatInsights/insights-schemas-java/blob/master/insights-notification-schemas-java/src/main/resources/schemas/Action.json)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)